### PR TITLE
Add Testing

### PR DESCRIPTION
**Pull request by thcdevcl**
Added testing to methods

### CODE REVIEW

File: spec/board_spec.rb [hunk 1]
```diff
@@ -0,0 +1,79 @@
+require './lib/board.rb'
+require './lib/player.rb'
+
+RSpec.describe Board do
+  describe "#draw?" do
+    it "check if the game is a draw" do
+    player1 =Player.new('❌')
+    player2 =Player.new('⭕️')
```
Comment (inhaq) [inline spec/board_spec.rb]: instead of initiating it locally again, kindly use `let`

File: spec/player_spec.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+require './lib/player.rb'
+
+RSpec.describe Player do
+  describe "#add" do
+    it "returns false if the game start" do
```
Comment (inhaq) [inline spec/player_spec.rb]: no test here, kindly add test

**Review by inhaq — CHANGES_REQUESTED** [review]
Good job, before proceeding to the next project, kindly do these improvements. 
We already know that unit testing is done for the function to see whether the input we're providing has the same expected output which we're expecting or not. Also, kindly check those functions separately, as you're using `make_play` and `board.draw?` in the same test. 
Once updated, kindly resubmit for review. If any questions, don't hesitate to reach me on slack.

File: spec/board_spec.rb (hunk 1 above)
Comment (thcdevcl) [reply spec/board_spec.rb]: Hi inhaq! So we cannot call a method inside a test that is not supossed to call that method? For example, we cannot call make_play on #draw? test and whats the best approach to solve this?

Comment (inhaq) [reply spec/board_spec.rb]: Hi @noincomedev, good question. You can use hard-coded input for the function to see what output will be. That's why it's not needed to call for make_play function again and again. I hope it makes sense.

**Pull request closed without merging**